src/engine/core/Application.cpp

```diff
@@ -85,6 +85,11 @@ namespace lei3d
 		}
 		glfwMakeContextCurrent(m_Window);
 
+		// Removes 60 fps cap set by GLFW without having to use a single buffer
+		// Apparently vendor/GPU specific if it works or not
+		// https://stackoverflow.com/questions/50412575/is-there-a-way-to-remove-60-fps-cap-in-glfw 
+		glfwSwapInterval(0);
```

Comment:
I'm not sure if we want to disable vsync. The fps cap problem was related to the implementation of FrameTick. If we set m_DesiredFPS to 60 it actually ran at 30 fps, so we had to hack it by setting it to 120 to get 60 fps.

We could add an option to enable/disable vsync though.

Reply:
ah ok, i thought the desired fps was 120.